content/backstage/plugins/buildkite.md

```diff
@@ -40,30 +34,39 @@ gettingStarted:
             Authorization: 
               $env: BUILDKITE_TOKEN
 
+  - intro: Add plugin to the list of your plugins
+    language: typescript
+    code: |
+      // packages/app/src/plugins.ts
+      export { plugin as Buildkite } from '@roadiehq/backstage-plugin-buildkite';
+
   - intro: Add plugin API to your Backstage instance.
     language: typescript
     code: |
-      // packages/app/src/components/catalog/EntityPage.tsx
-      import {
-        Router as BuildkiteRouter,
-        isPluginApplicableToEntity as isBuildkiteAvailable,
-      } from '@roadiehq/backstage-plugin-buildkite';
+     // packages/app/src/components/catalog/EntityPage.tsx
+     import {
+       EntityBuildkiteContent,
+       isPluginApplicableToEntity as isBuildkiteAvailable,
+     } from '@roadiehq/backstage-plugin-buildkite';
 
-      const CICDSwitcher = ({ entity }: { entity: Entity }) => {
-        switch (true) {
-          ...
-          case isBuildkiteAvailable(entity):
-            return <BuildkiteRouter entity={entity} />;
-          ...
-        }
-      }
+     const CICDSwitcher = ({ entity }: { entity: Entity }) => {
```

Comment:
This is a bit out of date. While it'll work we should push users to use the new catalog API. You can see an example in the upstream backstage (packages/app/src/components/catalog/EntityPage.tsx):
```
export const cicdContent = (
  <EntitySwitch>
    <EntitySwitch.Case if={isJenkinsAvailable}>
      <EntityJenkinsContent />
    </EntitySwitch.Case>

    <EntitySwitch.Case if={isBuildkiteAvailable}>
      <EntityBuildkiteContent />
    </EntitySwitch.Case>
    ```